x-pack/plugins/observability_solution/observability_ai_assistant/server/utils/recall/score_suggestions.ts

```diff
@@ -159,6 +148,6 @@ export async function scoreSuggestions({
 
   return {
     relevantDocuments,
-    scores: scores.map((score) => ({ id: score.originalId!, score: score.score })),
```

Comment:
I don't think `originalId` or other relics from `shortIdTable` should leak outside the communication with the LLM. It should be restricted to prompts and parsing replies.